source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -1519,28 +1501,26 @@ TEST_F(NiDAQmxDriverApiTests, AIVoltageChannel_CfgSampClkTimingAndAcquireData_Su
 TEST_F(NiDAQmxDriverApiTests, AIVoltageChannel_CfgSampClkTimingAndAcquireDataWithMinimumTimeout_SampleNotAvailableStatusIsReturned)
 {
   create_ai_voltage_chan(0.0, 1.0);
-  const auto TIMEOUT = 100.0;
+  const auto TIMEOUT = 100e-3;
   const auto SAMPLE_RATE = 1000.0;
   const auto SAMPLES_PER_CHAN = 60000UL;
   const auto NUM_SAMPS = 60000;
-  const auto EXPECTED_ERROR_STATUS = -200284;
 
   CfgSampClkTimingResponse response;
-  auto config_status = cfg_samp_clk_timing(
+  cfg_samp_clk_timing(
       create_cfg_samp_clk_timing_request(SAMPLE_RATE, Edge1::EDGE1_RISING, AcquisitionType::ACQUISITION_TYPE_FINITE_SAMPS , SAMPLES_PER_CHAN));
   start_task();
```

Comment:
Along with the suggested rename, move these to the "arrange" section.

(i.e. move them above the 1st blank line)